src/Castle.Core/Core/Logging/DiagnosticsLogger.cs

```diff
@@ -23,6 +23,9 @@ namespace Castle.Core.Logging
 	/// </summary>
 #if FEATURE_SERIALIZATION
 	[Serializable]
+#endif
+#if NET6_0_OR_GREATER
+	[System.Runtime.Versioning.SupportedOSPlatform("windows")]
 #endif
```

Comment:
Should this be `NET5_0_OR_GREATER` since `SupportedOSPlatformAttribute` was added in .NET 5? Obviously not that important because we don't build a .NET 5 build.

Reply:
I went with 6 because 5 was never targeted.